services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -1182,7 +1183,9 @@ public boolean finalizeVirtualMachineProfile(VirtualMachineProfile profile, Depl
         s_logger.debug(String.format("Setting UseHttpsToUpload config on cmdline with [%s] value.", useHttpsToUpload));
         buf.append(" useHttpsToUpload=").append(useHttpsToUpload);
 
-        addSecondaryStorageServerAddressToBuffer(buf, secStore, vmName);
+        for (DataStore secStore : secStores) {
+            addSecondaryStorageServerAddressToBuffer(buf, secStore, vmName);
```

Comment:
@shwstppr 
this will lead to multiple `secondaryStorageServerAddress=x.x.x.x` in the cmdline
we need to handle it in SSVM (probably in `secstorage.sh`)

Reply:
@weizhouapache I've made the changes to pass it as `secondaryStorageServerAddress=address1,address2...`
But I couldn't find any usage of `secondaryStorageServerAddress` in `secstorage.sh`
It is only used in ssvm-check.sh and I've added changes there.